### DISCUSSION TITLE
Need Help with another world claim

### DISCUSSION THREAD

**Discussion by Theerkas12** [GriefPrevention/GriefPrevention#1365] [Q&A] [answered]
Hello I am having problem to do claims in nether or End can you help guys how to fix that? I would be very happy

**Comment by bigpresh** [comment]
Not with that little information, no, clearly not.

**Comment by Jikoo** [comment, marked as answer]
You probably didn't [configure GP to enable claims](https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#limiting-which-worlds-griefprevention-runs-on) in those worlds.
